utils/getRoutes.js

```diff
@@ -11,3 +10,15 @@ export default async () => {
 
   return articles
 }
+
+export const getFeatureRoutes = async (params) => {
```

Comment:
where are we using `params`?

Reply:
We don't need params here. Forgot to remove.